db/blip_handler.go

```diff
@@ -1011,10 +1014,10 @@ func (bh *blipHandler) sendGetAttachment(sender *blip.Sender, docID string, name
 		return nil, err
 	}
 
-	lNum, metaLengthOK := meta["length"].(json.Number)
-	metaLength, err := lNum.Int64()
-	if err != nil {
-		return nil, err
+	lNum, metaLengthOK := meta["length"]
+	metaLength, ok := base.ToInt64(lNum)
+	if !ok {
+		return nil, fmt.Errorf("invalid attachment length found in meta")
```

Comment:
There doesn't seem to be a functional change in the above code  (other than error message)- what's the motivation for changing it?

Reply:
Reviewed with Sarath - makes sense for unit tests that aren't doing a decode with UseNumber.